### PR TITLE
Change all internal refereces for dss to css

### PR DESCRIPTION
**Pull request by rhatdan**
This is just a cleanup to refer to container-storage-setup (css) rather
then docker-storage-setup (dss)

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: container-storage-setup.sh [hunk 1]
```diff
@@ -34,8 +34,8 @@ STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
 STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
-PIPE1=/run/dss-fifo1
-PIPE2=/run/dss-fifo2
+PIPE1=/run/css-fifo1
+PIPE2=/run/css-fifo2
```
Comment (rhvgoyal) [inline container-storage-setup.sh]: This will create issues with multiple css instances running, right? We probably should introduce some locking so that only one instance of container-storage-setup is running at a time.

Comment (rhvgoyal) [inline container-storage-setup.sh]: This improvement should be a separate PR though.

Comment (rhatdan) [inline container-storage-setup.sh]: Can we just add the pid of the running container so that they are different?

Comment (rhvgoyal) [inline container-storage-setup.sh]: You mean create a new pair of pipes for each running instance? I think child needs to know in advance the name of pipe to wait on.

Comment (rhvgoyal) [inline container-storage-setup.sh]: Ok, we are passing the name of named fifo pipe to child. So yes, using pid of running process should be fine, I guess.  This is better than locking.

**Comment by rhvgoyal** [comment]
LGTM

**Comment by rhvgoyal** [comment]
@rh-atomic-bot r+ 14fac9e
